python/ray/tune/tuner.py

```diff
@@ -164,7 +164,7 @@ def fit(self) -> ResultGrid:
                 raise TuneError(
                     f"Tune run failed."
                     f'Please use tuner = Tuner.restore("'
-                    f'{self._local_tuner.experiment_checkpoint_dir}") to resume.'
+                    f'{self._local_tuner.experiment_checkpoint_dir()}") to resume.'
```

Comment:
Not directly related, but if @property doesn't work, can we use `get_experiment_checkpoint_dir` instead as a name? Otherwise it's confusing as we have the `experiment_checkpoint_dir` property (not as a function) in several other locations

Reply:
That's a good point!
Changed to `get_experiment_checkpoint_dir` instead.